neo/Consensus/ConsensusService.cs

```diff
@@ -184,8 +185,30 @@ private void InitializeConsensus(byte viewNumber)
             {
                 ChangeTimer(TimeSpan.FromMilliseconds(Blockchain.MillisecondsPerBlock << (viewNumber + 1)));
             }
+
+            // Try to speed up consensus if cached future payloads exists
+            if (hasFuturePayload)
+            {
+                TryToLoadFuturePayload(context.FuturePreparationPayloads);
+                TryToLoadFuturePayload(context.FutureCommitPayloads);
+                TryToLoadFuturePayload(context.FutureChangeViewPayloads);
+                TryToLoadFuturePayload(context.FutureRecoveryPayloads);
+            }
         }
 
+        private void TryToLoadFuturePayload(ConsensusPayload[] payloadsArray)
```

Comment:
`TryToConsumeFuturePayload`?

Reply:
Sounds good and precise, @shargon, since now it is consuming and setting to `null`.